hack/generator/pkg/codegen/pipeline_convert_allof_and_oneof_to_objects.go

```diff
@@ -118,63 +122,72 @@ type synthesizer struct {
 	defs         astmodel.Types
 }
 
-func (s synthesizer) oneOfObject(oneOf astmodel.OneOfType) (astmodel.Type, error) {
-	// If there's more than one option, synthesize a type.
-	// Note that this is required because Kubernetes CRDs do not support OneOf the same way
-	// OpenAPI does, see https://github.com/Azure/k8s-infra/issues/71
-	var properties []*astmodel.PropertyDefinition
-
-	propertyDescription := "Mutually exclusive with all other properties"
-	err := oneOf.Types().ForEachError(func(t astmodel.Type, i int) error {
-		prop, err := s.convertToOneOfProperty(i, t)
-		if err != nil {
-			return err
-		}
+type propertyNames struct {
```

Comment:
Is this a `propertyNames` or a `propertyName`?

Reply:
There’s two names for the property in here…

Reply:
I guess I was looking at it like there's one name, and some metadata (the `json` bit).
It's not a big deal though.